hello_world_attest_fn_call/README.md

```diff
@@ -190,14 +190,17 @@ To dive deeper, let's again look at the `run` target in the
 }
 ```
 
-The `enclave_attested_application_public_key` contains
-the `enclave_attestation` over the Blocky AS server public key. 
-The `function_calls` section contains the `transitive_attestation` over
-the function call. The `bky-as` CLI verifies the `enclave_attestation`, extracts
-the Blocky AS server public key and uses it to verify the
-`transitive_attestation` to extract the `claims`. You can learn more about this
-process in the
-[Attestations in the Blocky Attestation Service](https://blocky-docs.redocly.app/attestation-service/concepts#attestations-in-the-blocky-attestation-service) 
+The `enclave_attested_application_public_key` contains the `enclave_attestation`
```

Comment:
I added a bit about verification here, so that I can point to it in the on chain example

Reply:
OK, I have it updated, but let's review